README.adoc

```diff
@@ -542,6 +542,14 @@ image::https://lucid.app/publicSegments/view/43f2740c-2a7f-4b7f-909e-434a5bbe3fb
 
 See the link:{project_root}/parallel-consumer-examples/parallel-consumer-example-streams/src/main/java/io/confluent/parallelconsumer/examples/streams/StreamsApp.java[Kafka Streams example] project, and it's test.
 
+[[confluent-cloud]]
+=== Confluent Cloud
+
+. After https://kafka-tutorials.confluent.io/creating-first-apache-kafka-producer-application/confluent.html#provision-your-fully-managed-kafka-cluster-in-confluent-cloud[provisioning a Confluent Cloud cluster].
```

Comment:
@astubbs There's an open PR to Kafka Tutorials to change the name of this step's heading, so this link will break.  To future proof, I think it's better NOT to link to Kafka Tutorials but rather pull the text into here.

Reply:
ack. updated - check it out :)